test/viz/test_kuviz.py

```diff
@@ -60,3 +65,106 @@ def test_kuviz_validation_fails_without_url(self):
         carto_kuviz = CartoKuvizMock(name=name, url=None, password=None)
         with self.assertRaises(CartoException, msg='Error creating Kuviz. Something goes wrong'):
             _validate_carto_kuviz(carto_kuviz)
+
+
+class TestKuvizPublisher(unittest.TestCase):
+    def setUp(self):
+        self.username = 'fake_username'
+        self.api_key = 'fake_api_key'
+        self.context = ContextMock(username=self.username, api_key=self.api_key)
+
+    def test_kuviz_publisher_create_local(self):
+        source_1 = Source(build_geojson([-10, 0], [-10, 0]))
+        source_2 = Source(build_geojson([0, 10], [10, 0]))
+        layer_1 = Layer(source_1)
+        layer_2 = Layer(source_2)
+        map = Map([
```

Comment:
`map` is a Python builtin function, not a biggie to overwrite, just thought I'd flag it though

Reply:
:+1: